FAQ.md

```diff
@@ -81,3 +81,26 @@ If the client is trying to be defensive to nonconformant registries, and receive
 Mounting without having to specify `from`, also known as automatic mount origin discovery, requires the registry to determine whether or not a blob exists in any repository. 
 If the existence check for the blob is done first, an immediate failure will indicate the lack of presence of a blob. 
 On the other hand, if the registry needs to perform further work to determine if the blob can be accessed by the mounter, it could create an information disclosure risk, in leaking that presence of a blob with that digest in the registry.
+
+**Q: Why is support for new digest algorithms being added?**
```

Comment:
Did we add any new algorithms? sha512 was already on the image-spec list of valid algorithms: https://github.com/opencontainers/image-spec/blob/main/descriptor.md#digests

Reply:
support for sha512 in the dist-spec?

Reply:
But we don't define the list of supported algorithms in distribution-spec.